File: Indie Effects Git/Assets/IndieEffects/JS Classes/Outline.js
#pragma strict
import IndieEffects;

@script RequireComponent (IndieEffects)
@script AddComponentMenu ("Indie Effects/Outline")

var fxRes : IndieEffects;

var threshold : float;
private var blurMat : Material;
var outlineShader : Shader; 

function Start () {
    fxRes = GetComponent("IndieEffects");
    blurMat = new Material(outlineShader);
}
function Update () {
    blurMat.SetTexture("_MainTex", fxRes.RT);
    blurMat.SetFloat("_Treshold", threshold);
}
function OnPostRender () {
    FullScreenQuad(blurMat);
}

File: Indie Effects Git/Assets/IndieEffects/CSharp Classes/Outline.cs
using UnityEngine;

[RequireComponent(typeof( IndieEffects))]
[AddComponentMenu("Indie Effects/Outline C#")]
public class Outline : MonoBehaviour
{

    public IndieEffects fxRes ;

    public float threshold;
    private Material blurMat;
    public Shader outlineShader; 

    public void Start () {
        fxRes = GetComponent<IndieEffects>();
        blurMat = new Material(outlineShader);
    }
    public void Update () {
        blurMat.SetTexture("_MainTex", fxRes.RT);
        blurMat.SetFloat("_Treshold", threshold);
    }
    public void OnPostRender () {
        IndieEffects.FullScreenQuad(blurMat);
    }
}